aws/backing-services/aurora-postgres.tf

```diff
@@ -0,0 +1,77 @@
+# Don't use `admin`
+# ("MasterUsername admin cannot be used as it is a reserved word used by the engine")
+variable "POSTGRES_ADMIN_NAME" {
+  type        = "string"
+  description = "Postgres admin user name"
+}
+
+# Must be longer than 8 chars
+# ("The parameter MasterUserPassword is not a valid password because it is shorter than 8 characters")
+variable "POSTGRES_ADMIN_PASSWORD" {
+  type        = "string"
+  description = "Postgres password for the admin user"
+}
+
+variable "POSTGRES_DB_NAME" {
+  type        = "string"
+  description = "Postgres database name"
+}
+
+variable "POSTGRES_INSTANCE_TYPE" {
+  type        = "string"
+  default     = "db.r4.large"
```

Comment:
Should we default to something smaller here for the example? I worry people use this later and incur charges they didn't expect :) - if they forget to clean up. Maybe: db.t2.small or db.t2.medium?